src/index.js

```diff
@@ -246,19 +251,29 @@ module.exports = {
         if (Array.isArray(errors) && errors.length > 0) {
           allErrors.push({ path, url, errors });
         } else {
-          summaries.push({ path, url, summary: shortSummary });
+          data.push({ path, url, summary: shortSummary, results });
         }
       }
 
-      const { error, summary } = processResults({
-        summaries,
+      const { error, summary, extra_data } = processResults({
+        data,
         errors: allErrors,
         show,
       });
 
       if (error) {
         throw error;
       }
+
+      if (extra_data) {
+        const size = Buffer.byteLength(JSON.stringify(extra_data.reportHtml));
+        console.log(
+          `Report metrics collected: audited_uri: '${chalk.magenta(
+            extra_data.url || extra_data.path,
+          )}', html_report_size: ${chalk.magenta(size / 1024)} kbs`,
```

Comment:
[dust] Maybe `KB` instead of `kbs`? (although I think technically it should be KiB for /1024, and KB if /1000)

https://www.ibm.com/docs/en/storage-insights?topic=overview-units-measurement-storage-data